src/dev/flang/tools/Fuzion.java

```diff
@@ -690,8 +692,16 @@ private Runnable parseArgsAceMode(String[] args)
    */
   private Runnable parseArgsForBackend(String[] args)
   {
+    var argsLeft = args;
+
     for (var a : args)
       {
+        argsLeft = Arrays.copyOfRange(argsLeft, 1, argsLeft.length);
+        if (a.equals("--"))
```

Comment:
This should probably be 
```
  if (_backend.runsCode() && a.equals("--"))
```
since backends like `-c` should not allow to pass arguments.

Reply:
If this is used for some other purpose by some backend, that backend should add support for `--`, we could (then, not now), e.g., replace the predicate `runsCode()` by `usesDashDash()` or similar. Backends that do not have this should refuse `--`.

Reply:
Since this is the last argument anyway, the undefined backend can have `takesApplictionArgs()==true`. In a later PR, this might be cleaned up, e.g., by replacing the undefined backend by something like the default interpreter backend. The whole argument parsing code is quite messy...